_data/site_menu.yml

```diff
@@ -31,47 +31,58 @@ installation:
       path: "/docs/reference/latest/installing_on_scvmm/index.html"
       desc: "How to install and configure the ManageIQ Management Engine appliance on Microsoft System Center Virtual Machine Manager (SCVMM)"
 
-appliances:
-  title: Appliances
+upgrading:
+  title: Upgrading
   children:
-    - title: Backup and Restore
-      path: "/docs/reference/latest/appliances/backup_and_restore.html"
-      desc: "How to backup and restore appliances"
     - title: Upgrading
       path: "/docs/reference/latest/appliances/upgrading.html"
       desc: "How to upgrade appliances"
 
-configuration:
-  title: Configuration
-  children:
-    - title: Deployment Planning Guide
-      path: "/docs/reference/latest/deployment_planning_guide/index.html"
-      desc: "ManageIQ Deployment Planning Guide"
-    - title: General Configuration
-      path: "/docs/reference/latest/general_configuration/index.html"
-      desc: "ManageIQ General Configuration"
-    - title: High Availability Guide
-      path: "/docs/reference/latest/high_availability_guide/index.html"
-      desc: "ManageIQ High Availability Guide"
-    - title: Appliance Hardening Guide
-      path: "/docs/reference/latest/appliance_hardening_guide/index.html"
-      desc: "ManageIQ Appliance Hardening Guide"
-
```

Comment:
So Configuration moved under Reference https://github.com/ManageIQ/manageiq-documentation/pull/1806/files#diff-46686079121737af3636a6d1c95d842fc981118049ec6ece21ce578576eebd50R201 ?

Reply:
Yes, this aligns with how these topics are shown in downstream.